Data/DataModel/Team.cs

```diff
@@ -29,9 +29,18 @@ public class Team
 
         public virtual List<Submission> Submissions { get; set; }
 
+        /// <summary>
+        /// The number of hint coins this team currently has
+        /// </summary>
+
         /// <summary>
         /// The number of hint coins this team currently has
         /// </summary>
         public int HintCoinCount { get; set; }
+
+        /// <summary>
+        /// The number of hint coins this team currently has
```

Comment:
In the GitHub diff this comment is showing up as being identical to the one above - should this one say "The number of hint coins this team has used" ?

Reply:
Yep, copy/paste bug, will fix.